eclair-core/src/test/scala/fr/acinq/eclair/integration/ChannelIntegrationSpec.scala

```diff
@@ -623,9 +624,17 @@ class StandardChannelIntegrationSpec extends ChannelIntegrationSpec {
     bitcoinClient.publishTransaction(revokedCommitTx).pipeTo(sender.ref)
     sender.expectMsg(revokedCommitTx.txid)
     bitcoinClient.publishTransaction(htlcSuccess.head).pipeTo(sender.ref)
-    sender.expectMsg(htlcSuccess.head.txid)
+    sender.expectMsgType[Any] match {
+      case txid: TxId => assert(txid == htlcSuccess.head.txid)
+      // 3rd stage txs (txs spending htlc txs) are not tested if C publishes htlc-success before F
+      case Failure(e: JsonRPCError) => assert(e.error.message == "txn-mempool-conflict")
+    }
     bitcoinClient.publishTransaction(htlcTimeout.head).pipeTo(sender.ref)
-    sender.expectMsg(htlcTimeout.head.txid)
+    sender.expectMsgType[Any] match {
+      case txid: TxId => assert(txid == htlcTimeout.head.txid)
+      // 3rd stage txs (txs spending htlc txs) are not tested if C publishes htlc-fail before F
```

Comment:
```suggestion
      // 3rd stage txs (txs spending htlc txs) are not tested if C publishes the htlc-penalty transaction before F publishes its htlc-timeout
```

Reply:
Fixed in [fbf6939](https://github.com/ACINQ/eclair/pull/2871/commits/fbf6939fd83099f608f6a88c51a5428e01602cd6). I didn't notice I had written htlc-fail instead of htlc-timeout also. Thanks.